README.md

```diff
@@ -7,8 +7,11 @@ For more information, please refer to the [online documentation](https://aiida-c
 
 
 ## How to cite
-If you use the workflow of this package, please cite the [original paper (10.1038/s41524-021-00594-6)](https://doi.org/10.1038/s41524-021-00594-6).
-In addition, one should cite the quantum engines whose implementations are used.
+If you use the workflow of this package, please cite the paper in which the work is presented:
+
+> [S. P. Huber et al., npj Comput. Mater. 7, 136 (2021); doi:10.1038/s41524-021-00594-6](https://doi.org/10.1038/s41524-021-00594-6)
+
+In addition, if you run the common workflows, please also cite the quantum engine(s) that you will use. We provide below a table of references for your convenience.
```

Comment:
Should we also add a reference to the AiiDA paper itself?

Reply:
Good point :-) I'll add it. I'll also add it to the docs (I just pushed a commit to fix also the docs).
At the end the commits can be squashed.

Reply:
I don't think I see the citation yet for the `aiida-core` paper?

Reply:
No I had only added some citations to the paper, before your comment. I just added it.